setup.py

```diff
@@ -60,7 +60,8 @@
     print('Adding extra setuptools args')
     extra_setuptools_args = dict(
         zip_safe=False,  # the package can run out of an .egg file
-        include_package_data=True,
+        include_package_data=False,
```

Comment:
Is there a particular reason for this? If we don't include package data, the `MANIFEST` (built from `sdist`) won't place the `.pyx` & `.c` files. But I guess we don't need those distributed... just for development. So maybe I answered my own question.